### PR TITLE
Delete commented out imports of deprecated JetCorrector cffs

### PR DESCRIPTION
**Pull request by wddgit**
#### PR description:

We just removed central cff files related to deprecated JetCorrector's (#39953). This PR removes lines of Python configuration code where those files were imported and those particular lines were already commented out.

@vlimant suggested we do this centrally in Issue #40304. Easy to do, so I just did it.

#### PR validation:

Only deletes comment lines.

### CODE REVIEW

**Comment by wddgit** [comment]
please test

**Comment by vlimant** [comment]
+1

File: PhysicsTools/TagAndProbe/test/Electron_TagProbeTreeProducer_cfg.py [hunk 1]
```diff
@@ -389,7 +389,6 @@
 ##   |_____/_/\_\\__\___|_|  |_| |_|\__,_|_|    \_/ \__,_|_|  |___/
 ##   
 ## Here we show how to use a module to compute an external variable
-## process.load("JetMETCorrections.Configuration.DefaultJEC_cff")
 ## ak5PFResidual.useCondDB = False
```
Comment (perrotta) [inline PhysicsTools/TagAndProbe/test/Electron_TagProbeTreeProducer_cfg.py:392]: This modifies a module from the line removed, can also get removed

File: L1Trigger/L1TNtuples/test/test_JetRecoProducer.py [hunk 2]
```diff
@@ -43,7 +43,6 @@
 process.load("L1Trigger.L1TNtuples.l1JetRecoTreeProducer_cfi")
 
 # get corrected jets
```
Comment (perrotta) [inline L1Trigger/L1TNtuples/test/test_JetRecoProducer.py:45]:
```suggestion
```
Also this comment can now get removed

File: Validation/RecoJets/python/JetValidation_cff.py [hunk 3]
```diff
@@ -18,7 +18,6 @@
     newAk4CaloL2L3Corrector
 )
 
-#from JetMETCorrections.Configuration.JetCorrectionServicesAllAlgos_cff import ak7CaloL2L3,ak7CaloL2Relative,ak7CaloL3Absolute
 #newAk7CaloL2L3 = ak7CaloL2L3.clone()
```
Comment (perrotta) [inline Validation/RecoJets/python/JetValidation_cff.py:21]:
```suggestion
```

File: Validation/RecoJets/python/JetValidation_cff.py [hunk 4]
```diff
@@ -29,7 +28,6 @@
     newAk4PFL1FastL2L3Corrector
 )
 
-#from JetMETCorrections.Configuration.JetCorrectionServices_cff import ak4JPTL1FastL2L3,ak4JPTL1Fastjet,ak4JPTL2Relative,ak4JPTL3Absolute
 #newAk4JPTL1FastL2L3 = ak4JPTL1FastL2L3.clone()
```
Comment (perrotta) [inline Validation/RecoJets/python/JetValidation_cff.py:31]:
```suggestion
```

File: L1Trigger/L1TNtuples/test/test_JetRecoProducer.py (hunk 2 above)
Comment (wddgit) [reply L1Trigger/L1TNtuples/test/test_JetRecoProducer.py]: Done. Thanks.

File: PhysicsTools/TagAndProbe/test/Electron_TagProbeTreeProducer_cfg.py (hunk 1 above)
Comment (wddgit) [reply PhysicsTools/TagAndProbe/test/Electron_TagProbeTreeProducer_cfg.py]: Done. Thanks.

File: Validation/RecoJets/python/JetValidation_cff.py (hunk 3 above)
Comment (wddgit) [reply Validation/RecoJets/python/JetValidation_cff.py]: Done. Thanks.

File: Validation/RecoJets/python/JetValidation_cff.py (hunk 4 above)
Comment (wddgit) [reply Validation/RecoJets/python/JetValidation_cff.py]: Done. Thanks.

**Comment by wddgit** [comment]
please test

Made all changes requested in comments, deleted more unneeded comment lines.

**Comment by perrotta** [comment]
+1
- This PR just removed some obsolete and already commented out lines: let merge it as such, to clean up the queue
- I hope that @cms-sw/dqm-l2 @cms-sw/l1-l2 @cms-sw/reconstruction-l2 @cms-sw/xpog-l2 have nothing in contrary: otherwise just speak up

**Comment by perrotta** [comment]
merge

**Comment by mandrenguyen** [comment]
+1

**Comment by emanueleusai** [comment]
+1

**Pull request merged**